momento-provider/src/main/java/com/google/code/ssm/providers/momento/MomentoClientWrapper.java

```diff
@@ -89,7 +90,7 @@ public boolean delete(final String key) throws CacheException {
                     cachedObject.getFlags(),
                     cachedObject.getData()
             );
-            accessLogWrite("Delete", key, result, cachedObject.getData().length);
+            accessLogWrite("Delete", key, result, cachedObject.getData().length, 1);
```

Comment:
what's this literal `1` for?

Reply:
Thats the TTL for the "delete" api call right now. Since way we are "deleting" right now is set key to empty string w/ ttl 1. Its something we need to fix when we get proper delete api this is just for the debug logging here.

Reply:
ah i see